.github/workflows/commit-verifier.yaml

```diff
@@ -36,3 +39,12 @@ jobs:
             echo "For more details, see .github/workflows/commit-message.yaml"
             exit 1
           }
+
+      - name: Verify GPG Signature
+        run: |
+            LAST_COMMIT=$(git rev-parse HEAD)
```

Comment:
Since this only looks at the last commit, what if I make many commits locally and only sign the most recent one, then push. In that case this check will pass from my understanding? Is that a problem?

Reply:
Good catch. I'll remove this altogether. I think for now we can just update the docs which was the original intention of this PR.